vpr/src/base/vpr_api.cpp

```diff
@@ -845,8 +841,10 @@ void vpr_init_graphics(const t_vpr_setup& vpr_setup, const t_arch& arch) {
 }
 
 void vpr_close_graphics(const t_vpr_setup& /*vpr_setup*/) {
+#ifndef NO_GRAPHICS
```

Comment:
Looks like this is the only one left out side of vpr/draw. Can it also be removed?